python/interpret_community/widget/explanation_dashboard.py

```diff
@@ -226,14 +221,18 @@ def _local_port_available(port, rais=True):
             return True
 
     def __init__(self, explanation, model=None, *, dataset=None,
-                 true_y=None, classes=None, features=None, port=None, use_cdn=True,
-                 datasetX=None, trueY=None, locale=None, public_ip=None, with_credentials=False):
+                 true_y=None, classes=None, features=None, port=None,
+                 datasetX=None, trueY=None, locale=None, public_ip=None,
+                 with_credentials=False, use_cdn=None):
+        if use_cdn is not None:
+            warnings.warn("""use_cdn parameter is deprecated, cdn has been deleted.
```

Comment:
should we add doc string and mark it deprecated on top of run time warning?

Reply:
done